packages/lib-utils/src/k8s/hooks/index.ts

```diff
@@ -0,0 +1 @@
+export { useK8sWatchResource } from './useK8sWatchResource';
```

Comment:
@vojtechszocs I'm pretty sure you said I don't need to export types used -- is that true or should I be exporting the type behind this?

Reply:
@andrewballantyne This ultimately depends on whether we want to have a direct export of `UseK8sWatchResource` type in `packages/lib-utils/src/index.ts` which is the main module of the library.

Taking `packages/lib-core/src/index.ts` as an example, we currently don't export dedicated hook function types, so I'd add it later if consumers need to reference the type directly.

Reply:
> don't need to export types used

When a function `foo` refers to type `X` even if type `X` isn't directly exported in the main module, it's still included in the generated `dist/index.d.ts` typings bundle.

The decision whether or not to export `X` directly depends on whether or not consumers need to reference it directly.

Reply:
Right, I get that ... but that's only helpful when we are talking docs... for other TS usages, it won't be available. Hence why I pushed out the Model Common and Resource Common types in my other PR -- those need to be used by the caller/user of the SDK.